docs/docs/workflows/config.md

```diff
@@ -0,0 +1,228 @@
+# Configure your cluster
+
+Before your cluster can be created, you need to configure the identity and access management (IAM) for your CSP and choose machine types for your cluster.
+
+## Choosing VM types
+
+Constellation supports the following VM types:
+<tabs groupID="csp">
+<tabItem value="azure" label="Azure">
+
+By default, Constellation uses `Standard_DC4as_v5` CVMs (4 vCPUs, 16 GB RAM) to create your cluster. Optionally, you can switch to a different VM type by modifying **instanceType** in the configuration file. For CVMs, any VM type with a minimum of 4 vCPUs from the [DCasv5 & DCadsv5](https://docs.microsoft.com/en-us/azure/virtual-machines/dcasv5-dcadsv5-series) or [ECasv5 & ECadsv5](https://docs.microsoft.com/en-us/azure/virtual-machines/ecasv5-ecadsv5-series) families is supported.
+
+You can also run `constellation config instance-types` to get the list of all supported options.
+
+</tabItem>
+<tabItem value="gcp" label="GCP">
+
+By default, Constellation uses `n2d-standard-4` VMs (4 vCPUs, 16 GB RAM) to create your cluster. Optionally, you can switch to a different VM type by modifying **instanceType** in the configuration file. Supported are all machines from the N2D family. Refer to [N2D machine series](https://cloud.google.com/compute/docs/general-purpose-machines#n2d_machines) or run `constellation config instance-types` to get the list of all supported options.
+
+</tabItem>
+<tabItem value="aws" label="AWS">
+
+By default, Constellation uses `m6a.xlarge` VMs (4 vCPUs, 16 GB RAM) to create your cluster. Optionally, you can switch to a different VM type by modifying **instanceType** in the configuration file. Supported are all nitroTPM-enabled machines. Refer to the [list of nitroTPM-enabled instance types](https://docs.aws.amazon.com/AWSEC2/latest/UserGuide/enable-nitrotpm-prerequisites.html) or run `constellation config instance-types` to get the list of all supported options.
+
+</tabItem>
+</tabs>
+
+Fill the desired VM type into the **instanceType** field in the `constellation-conf.yml` file.
+
+## Creating an IAM configuration
+
+IAM configuration for your cluster can be created automatically using the `constellation iam create` command.
+<tabs groupId="csp">
+<tabItem value="azure" label="Azure">
+
+You must be authenticated with the [Azure CLI](https://learn.microsoft.com/en-us/cli/azure/install-azure-cli) in the shell session.
+
+```bash
+constellation iam create azure --region=westus --resourceGroup=constellTest --servicePrincipal=spTest
+```
+
+:::info
+This command creates IAM configuration on the Azure region `westus` creating a new resource group `constellTest` and a new service principal `spTest`.
+:::
+
+Note that CVMs are currently only supported in a few regions, check [Azure's products available by region](https://azure.microsoft.com/en-us/global-infrastructure/services/?products=virtual-machines&regions=all). These are:
+  * `westus`
+  * `eastus`
```

Comment:
IMO the regions are essential enough to also mention them in the *first steps*. We should prevent users from making a mistake there.